app/models/supervisor_script.rb

```diff
@@ -16,41 +18,27 @@ def self.collection_name
     'supervisor_scripts'
   end
 
-  SM_PATH = 'scalarm_supervisor_scrpits/simulated_annealing/anneal.py'
-  LOG_FILE_PREFIX = 'log/supervisor_script_log_'
-  LOG_FILE_SUFFIX = '.log'
-  CONFIG_FILE_PREFIX = '/tmp/supervisor_script_config_'
-
   ##
-  # Starts new supervised script (for now only simulated annealing one). Performed actions:
+  # Starts new supervised script. Performed actions:
   # * Gets Experiment Manager Address from Information Service
-  # * Creates config file in /tmp/supervisor_script_config_<experiment_id>
-  # * Starts supervisor script with output set to log/supervisor_script_log_<experiment_id>
+  # * Runs supervisor script using proper executor
   #
   # Required params
-  # * id - id of supervisor script (for now only placeholder)
+  # * id - id of supervisor script
   # * config - json with config for supervisor script (config is not validated)
   # Returns
-  # * pid of startes script
+  # * pid of started script
   # Raises
   # * Various StandardError exceptions caused by creating file or process starting.
```

Comment:
żeby było konsekwentnie to
creating file -> file creation
albo
process starting -> starting process